src/SwedbankPay/Api/Client/ClientVersion.php

```diff
@@ -0,0 +1,197 @@
+<?php
+
+namespace SwedbankPay\Api\Client;
+
+use \SwedbankPay\PathResolver as PathResolver;
+use \SwedbankPay\Api\Client\Exception as ClientException;
+
+/**
+ * Class ClientVersion
+ * @package SwedbankPay\Api\Client
+ */
+class ClientVersion
+{
+    private $clientVersion;
+
+    /**
+     * Version constructor
+     *
+     * @throws ClientException
+     */
+    public function __construct()
+    {
+        $this->clientVersion = $this->getVersionFromEnvironment();
+    }
+
+    /**
+     * Gets the version number from a defined constant VERSION, the environment
+     * variable VERSION or from composer.json.
+     *
+     * @return string Version number
+     */
+    public function getVersion() : string
+    {
+        return $this->clientVersion;
+    }
+
+    /**
+     * Gets the version number from a defined constant VERSION, the environment
+     * variable VERSION or from composer.json.
+     *
+     * @return string Version number
+     * @throws ClientException
+     */
+    protected function getVersionFromEnvironment() : string
+    {
+        $version = null;
+
+        if ($this->tryGetVersionNumberFromConstant($version)) {
+            return $version;
+        }
+
+        if ($this->tryGetVersionNumberFromEnv($version)) {
+            return $version;
+        }
+
+        if ($this->tryGetVersionNumberFromComposerJson($version)) {
+            return $version;
+        }
+
+        if ($this->tryGetVersionNumberFromComposerLock($version)) {
+            return $version;
+        }
+
+        throw new ClientException('VERSION not found in environment variable, composer.json or anywhere else.');
+    }
+
+
+    /**
+      * Tries to get the version number from the constant VERSION.
+      * Returns true if successful; otherwise false.
+      *
+      * @param string $version The by-reference $version variable to assign the version number to, if found.
+      * @return bool true if successful; otherwise false.
+      */
+    private function tryGetVersionNumberFromConstant(&$version) : bool
+    {
+        if (defined('VERSION') && !is_null(VERSION) && !empty(VERSION)) {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The use of function is_null() is discouraged; use strict comparison "=== null" instead.](https://app.codacy.com/gh/SwedbankPay/swedbank-pay-sdk-php/pullRequest?prid=5312212)